packages/ipfs/src/core/components/start.js

```diff
@@ -130,7 +138,6 @@ module.exports = ({
     apiManager.update(api, () => undefined)
   } catch (err) {
     cancel()
-    startPromise.reject(err)
```

Comment:
Why did this get removed?  If the user calls `node.start()` twice, the second promise will never resolve/reject.

I'm not sure how common this is, but I'd like to re-evaluate the use of the api proxy as part of #2762.

Reply:
Oh, I did not understand that this was used to restart.

So, what happens with the `startPromise.reject(err)` is that this promise rejection is not being handled anywhere. I looked closely on this now, and it seems that the `ApiManager` should take care of handling promise rejections. Is that fair to assume? I am not sure on how to actually do this though.

Reply:
It's not used to restart, it's if the user calls `node.start()` on a node that's already starting up.

Reply:
sorry, I understood, but misspelled it: parallel starts

So, how to you think we should go on handling the promise rejections on the APIManager

Reply:
I added a catch, but I am not sure this is the best approach here. Let me know